File: DouYu/app/src/test/java/cn/edu/app/douyu/core/MoneyFormatterTest.java
package cn.edu.app.douyu.core;

import org.junit.Test;

import static org.junit.Assert.assertEquals;

public class MoneyFormatterTest {
    @Test
    public void formatsCentsAsYuan() {
        assertEquals("¥0.00", MoneyFormatter.centsToYuan(0));
        assertEquals("¥12.34", MoneyFormatter.centsToYuan(1234));
        assertEquals("--", MoneyFormatter.centsToYuan(null));
    }
}

File: DouYu/app/src/test/java/cn/edu/app/douyu/core/MoneyFormatterTest.kt
package cn.edu.app.douyu.core

import cn.edu.app.douyu.core.ui.formatPriceCent
import org.junit.Assert.assertEquals
import org.junit.Test

class MoneyFormatterTest {
    @Test
    fun formatsCentAmountAsChineseYuan() {
        assertEquals("¥29.90", formatPriceCent(2990))
        assertEquals("¥0.00", formatPriceCent(0))
        assertEquals("¥128.05", formatPriceCent(12805))
    }
}
